### PR TITLE
Code clean for schedule util.

### PR DESCRIPTION
**Pull request by yanggangtony**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
1 Code clean for ```GetPodStartTime(pod) and corev1helpers.PodPriority(pod)```.
2 golint tips:  `Yoda Conditions: To Yoda or Not to Yoda `, for the ```if string(patchBytes) == "{}" ``` 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: pkg/scheduler/util/utils.go [hunk 1]
```diff
@@ -67,13 +67,15 @@ func GetEarliestPodStartTime(victims *extenderv1.Victims) *metav1.Time {
 	maxPriority := corev1helpers.PodPriority(victims.Pods[0])
 
 	for _, pod := range victims.Pods {
-		if corev1helpers.PodPriority(pod) == maxPriority {
-			if GetPodStartTime(pod).Before(earliestPodStartTime) {
-				earliestPodStartTime = GetPodStartTime(pod)
+		earliestPodStartTimeTmp := GetPodStartTime(pod)
+		maxPriorityTmp := corev1helpers.PodPriority(pod)
```
Comment (sanposhiho) [inline pkg/scheduler/util/utils.go:71]: please remove Tmp at the end on both.

**Review by sanposhiho — COMMENTED** [review]
one nit

File: pkg/scheduler/util/utils.go (hunk 1 above)
Comment (sanposhiho) [reply pkg/scheduler/util/utils.go:71]: Nah, ok then that'll result in the conflict with existing vars. Sorry, nvm

**Review by sanposhiho — APPROVED** [review]
/lgtm

File: pkg/scheduler/util/utils.go (hunk 1 above)
Comment (ahg-g) [reply pkg/scheduler/util/utils.go:71]: https://github.com/kubernetes/kubernetes/pull/117390 does it in a better way, lets merge that one.

**Pull request closed without merging**